pgcontents/tests/test_synchronization.py

```diff
@@ -214,6 +214,13 @@ def encrypted_pgmanager(user_id):
         paths = [(user_id, path)
                  for user_id in user_ids
                  for path in populate(managers[user_id])]
+
+        # Create a text file for each user as well, which should be ignored by
+        # the notebook generators
+        model = {'content': 'text file contents', 'format': 'text'}
+        for manager in managers.values():
+            manager.new(model, path='text file.txt')
```

Comment:
(apparently in python3 there's `assertLogs` for this kind of testing https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs)